src/toolkits/activity_classification/activity_classifier.cpp

```diff
@@ -859,8 +864,12 @@ void activity_classifier::perform_training_iteration() {
   const size_t iteration_idx = read_state<flex_int>("training_iterations");
 
   float cumulative_batch_loss = 0.f;
-  float cumulative_batch_accuracy = 0.f;
+  // float cumulative_batch_accuracy = 0.f;
   size_t num_batches = 0;
+  size_t cumulative_train_chunk = 0;
+  size_t cumulative_train_chunks = 0;
+  size_t num_train_samples = 0;
+  size_t total_train_samples = 0;
```

Comment:
Same as above, let's move two of these into the lambda. And maybe `{train/batch}_num_{correct/samples}`